lib/ops/opsservice/configure.go

```diff
@@ -1190,9 +1191,17 @@ func (s *site) getTeleportNodeConfig(ctx *operationContext, masterIPs []string,
 		fileConf.Logger.Severity = "info"
 	}
 
+	// Add all available master nodes as auth servers for the teleport node.
 	for _, masterIP := range masterIPs {
-		fileConf.AuthServers = append(fileConf.AuthServers, fmt.Sprintf("%v:3025", masterIP))
+		fileConf.AuthServers = append(fileConf.AuthServers,
```

Comment:
Is this an issue when all master servers have been replaced but each node still has their IP addresses in configuration? Will this impact performance? Maybe leave just the node port service as the sole source of truth?

Reply:
No it won't be an issue, teleport will try all listed servers, there shouldn't be performance impact. I thought about just using node port, but decided against it to make sure teleport nodes can still connect to auth server in case service network is non-functioning.